FreeITCourse/src/lesson05/MetodTask21.java

```diff
@@ -0,0 +1,49 @@
+package lesson05;
+
+
+import java.util.Date;
+
+
+// Задание 21
+// Напишите три цикла выполняющих многократное сложение строк,
+// один с помощью оператора сложения и String,
+// другой с помощью StringBuilder и метода append,
+// а также аналогино для StringBuffer.
+// Сравните скорость их выполнения.
+public class MetodTask21 {
+    public static void main(String[] args) {
+// check String +
+
+        String str = "Check  of add";
+        String addStr = " string ";
+        Date date1 = new Date();
+        for (int i = 0; i < 30000 ; i++) {
+            str += addStr;
+        }
+        Date date2 = new Date();
+        long result1 = date2.getTime() - date1.getTime();
+        System.out.println("Total time for String + equals: " + result1 + " milisecond(s)");
+
+// check StringBuilder
+
+        StringBuilder strB = new StringBuilder("Check  of add");
```

Comment:
no need to cut variable name in this case. stringBuilder will be better

Reply:
corrected